src/main/java/uk/gov/digital/ho/hocs/info/api/CountryResource.java

```diff
@@ -27,6 +33,34 @@ public CountryResource(CountryService countryService) {
         return ResponseEntity.ok(countrys.stream().map(Country::getName).collect(Collectors.toSet()));
     }
 
+    @GetMapping(value = "/countries/local", produces = APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<?> getAllActiveCountries() {
```

Comment:
I don't understand these two: getAllActiveCountries and getAllActiveTerritories
Looks like they are to replace the existing endpoint by loading data in from files rather than DB, but the original endpoint returned both countries and territories combined.

Reply:
@abarker66 Can you shed any light on this? Where are the endpoints used form? I guess its been regression tested?

Reply:
Anyway, it's not a merge problem. I'll take a look at countries separately.